src/hooks/Makefile.am

```diff
@@ -36,10 +36,9 @@
 # trademark licensing policies.
 #
 
-pbshooksdir = $(libdir)/@PBS_PYTHON_DESTLIB@/pbs_hooks
+pbshooksdir = $(libdir)/python/altair/pbs_hooks
```

Comment:
Since the version of Python PBS relies on is controlled by the spec file, we'll have to ensure that all hooks are compatible. This is nothing new, but this change does prevent maintaining multiple versions of hooks in the future. That's probably best, just pointing it out.